src/avatar_action.cpp

```diff
@@ -1033,7 +1033,16 @@ void avatar_action::use_item( avatar &you, item_location &loc )
         // Adjustment because in player::wield_contents this amount is refunded.
         you.mod_moves( -loc.obtain_cost( you ) );
     } else {
+        item_location::type loc_where = loc.where();
+        if( loc_where != item_location::type::character && loc_where != item_location::type::container ) {
```

Comment:
Does `item_location::type::container` mean it's in a container on the character, or can it be in a container on the map/in a vehicle, too?

Reply:
Good catch, just tested and yea that is a mistake I will have to amend.

Reply:
If that's the case, `container` should not be part of that enum. Not as part of this PR obviously, but it would make a fix for this not awkward.

Reply:
It's a bit different actually. But I made #46939 that you can make use of if it gets merged.

Reply:
That sounds perfect, I'll be sure to add it to this once it merges.  Thanks